lib/db/schema/patch-083-084.sql

```diff
@@ -0,0 +1,57 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('83');
+
+-- Since we are altering the size of the `recoveryKeyId` column
+-- we need to make sure that the column is empty before it can be applied
+DELETE from recoveryKeys;
```

Comment:
This table is safe to clear since the UX to create recovery keys has not landed. Anything in here would have to be test data.

Reply:
Sounds good, but I think it's worth adding that additional explanation into the comment in the file itself for our future reference.